bbp/modules/default.nix

```diff
@@ -2518,6 +2518,15 @@ let
         };
 
 
+	java = pkgs.envModuleGen rec {
+		name = "java";
+		version = "1.8";
+            	moduleFilePrefix = "nix/py27";
```

Comment:
@adevress Are you sure about this `moduleFilePrefix` ?

Reply:
@tristan0x  Fixed, thank you Tritri !

Peace, love, Chocolate and coeur, coeur, coeur